tests/cql_query_test.cc

```diff
@@ -1632,6 +1637,35 @@ SEASTAR_TEST_CASE(test_aggregate_functions) {
             net::inet_address("1::1"),
             net::inet_address("1.0.0.1")
         ).test_min_max_count();
+
+        auto list_type_int = list_type_impl::get_instance(int32_type, false);
+        aggregate_function_test(e, list_type_int,
+            make_list_value(list_type_int, {1, 2, 3}),
+            make_list_value(list_type_int, {1, 2, 4}),
+            make_list_value(list_type_int, {2, 2, 3})
+        ).test_min_max_count();
+
+        auto set_type_int = set_type_impl::get_instance(int32_type, false);
+        aggregate_function_test(e, set_type_int,
+            make_set_value(set_type_int, {1, 2, 3}),
+            make_set_value(set_type_int, {1, 2, 4}),
+            make_set_value(set_type_int, {2, 3, 4})
+        ).test_min_max_count();
+
+        auto tuple_type_int_text = tuple_type_impl::get_instance({int32_type, utf8_type});
+        aggregate_function_test(e, tuple_type_int_text,
+            make_tuple_value(tuple_type_int_text, {1, "aaa"}),
+            make_tuple_value(tuple_type_int_text, {1, "bbb"}),
+            make_tuple_value(tuple_type_int_text, {2, "aaa"})
+        ).test_min_max_count();
+
+        auto map_kv_pair = tuple_type_impl::get_instance({int32_type, utf8_type});
```

Comment:
Is it a leftover from previous iterations? This type doesn't seem very useful, you could just use int32_type and utf8_type directly in `map_type_impl::get_instance`. I'm fine with this code as is, but if you happen to fix the order of `inline` keywords above, you could also tackle this.

Reply:
done!